package.json

```diff
@@ -32,10 +32,10 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-react": "^7.4.0",
     "eslint-plugin-standard": "^3.0.1",
-    "faucet-pipeline": "^0.14.0",
-    "faucet-pipeline-js": "^0.15.4",
-    "faucet-pipeline-sass": "^0.10.0",
-    "faucet-pipeline-static": "^0.7.0",
+    "faucet-pipeline": "^1.0.0-beta.0",
+    "faucet-pipeline-js": "^1.0.0-beta.0",
+    "faucet-pipeline-sass": "^1.0.0-beta.0",
+    "faucet-pipeline-static": "^1.0.0-beta.0",
```

Comment:
I failed to use Yarn for installing these, so you'll wanna fix `yarn.lock` yourself.